scifight/admin.py

```diff
@@ -1,3 +1,5 @@
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import User
```

Comment:
I'd better write `from django.contrib import auth` and use names like `auth.models.User` and `auth.admin.UserAdmin` to distinguish them from the classes you're created here to override them.